eng/empty.csproj

```diff
@@ -10,6 +10,7 @@
 
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
+    <AssemblyName>.NET Monitor</AssemblyName>
```

Comment:
So does this value become the original file name and the file description?

Reply:
Correct

Reply:
This generation could be done on a per-file basis so that the description and original name are more tied to the file, but that would take a lot more work than this initial cleanup and inclusion.

Reply:
Did you want to use dotnet-monitor or perhaps dotnet monitor?

Reply:
Now using `dotnet-monitor`